examples/awesome-vault-service/config/config.toml

```
@@ -27,3 +19,22 @@ aggregator_server_ip_port = "localhost:8090"
 task_manager_address = "0x7bc06c482dead17c0e297afbc32f6e63d3846650"
 
 operator_state_retriever_address = "0x5f3f1dbd7b74c6b46e8c44f98792a1daf8d69154"
+
+[Registration]
```

Let's make this lowercase too

```suggestion
[registration]
```

---

Done in [d5f6a6d](https://github.com/Layr-Labs/eigensdk-go/pull/711/commits/d5f6a6dbc52d9311019b13ba3364f13f29bdbdcf)